modules/bbp-simulation/src/main/resources/schemas/hbphackaton/simulation/analysis/v0.1.0.json

```diff
@@ -0,0 +1,56 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
+    {
+      "nsg": "https://bbp-nexus.epfl.ch/vocabs/bbp/neurosciencegraph/core/v0.1.0/",
+      "this": "{{base}}/schemas/hbphackaton/simulation/analysis/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/bbp/core/activity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:AnalysisShape",
+      "@type": "sh:NodeShape",
+      "label": "Model or simulation report analysis shape",
+      "comment": "The analysis strategy/protocol is defined in a protocol.",
+      "targetClass": "nsg:Analysis",
```

Comment:
Would a more specific type be more appropriate to use instead of **Analysis**? Analysis could be the super type.

Reply:
This is the super type. We have PSTH plot, raster plot,,... all kind of analysis. Do we really need a entity for each of them ?